core/src/consensus/tendermint/mod.rs

```diff
@@ -762,6 +762,18 @@ impl ConsensusEngine<CodeChainMachine> for Tendermint {
     fn register_chain_notify(&self, client: &Client) {
         client.add_notify(Arc::downgrade(&self.chain_notify) as Weak<ChainNotify>);
     }
+
+    fn get_block_hash_to_mine_on(&self, _best_block_hash: H256) -> H256 {
+        let c = match self.client.read().as_ref().and_then(|weak| weak.upgrade()) {
+            Some(c) => c,
+            None => return Default::default(),
```

Comment:
When does it happen?
And is it okay just return zero hash?

Reply:
This could happen when the codechain exits.
The program will be killed soon, so returning default is ok.

Reply:
If then, how about using `expect` so it will be killed or returning `Option`?
I want to make the meaning clear.

Reply:
Ok.